datacube/storage/_read.py

```diff
@@ -188,9 +187,21 @@ def norm_read_args(roi, shape, extra_dim_index):
             warp_affine(pix, dst, A, resampling,
                         src_nodata=rdr.nodata, dst_nodata=dst_nodata)
         else:
+            # XSCALE and YSCALE are (currently) undocumented arguments that rasterio passed through to
```

Comment:
This applies equally to `warp_affine` as it calls the same gdal function under the hood.

Reply:
Fixed.